bodhi/server/consumers/updates.py

```diff
@@ -36,7 +36,7 @@
 
 import fedmsg.consumers
 
-from bodhi.server.bugs import bugtracker
+from bodhi.server import bugs as bug_module
```

Comment:
@pypingou I did this because there are already variables used in this module called ```bugs```, and it was easier to import this under a different name than to rename all those variables. The variables actually have a more natural usage anyway (such as ```for bug in bugs```) and the bugs module isn't so natural (it doesn't actually contain bugs, it just has a class that can help you query bugzilla.)